src/gui/fullinspector.h

```diff
@@ -50,21 +50,28 @@ public slots:
     void slotDeselectedNode(const Node& node);
     void slotDeselectedEdge(const Edge& edge);
     void slotDelete();
-    void slotChangeAttrScope(AttributesScope nodeAttrScope);
+    void slotChangeNodeAttrsScope(AttributesScope attrScope);
+    void slotChangeEdgeAttrsScope(AttributesScope attrScope);
 
 signals:
     void deleteNodes();
+    void deleteEdges();
 
 private:
     QWidget* m_parent;
     Ui_FullInspector* m_ui;
     std::map<int, Node> m_selectedNodes;
     std::map<int, Edge> m_selectedEdges;
     std::vector<std::shared_ptr<AttrWidget>> m_nodeAttrWidgets;
+    std::vector<std::shared_ptr<AttrWidget>> m_edgeAttrWidgets;
 
-    void attrValueChanged(int attrId) const;
+    void attrNodeValueChanged(int attrId) const;
+    void attrEdgeValueChanged(int attrId) const;
     void hideLayout(QFormLayout* layout);
     void showLayout(QFormLayout* layout);
+
+private slots:
+    void slotChangeAttrScope(AttributesScope nodeAttrScope, std::vector<std::shared_ptr<AttrWidget>>& attrWidget, QFormLayout* lattrs);
```

Comment:
it doesn't need to be a slot
it could be just a private function... might be better to also rename it to something like `updateAttrScope`
the parameter `AttributesScope nodeAttrScope` should be `const AttributesScope& attrScope`